packages/stark-ui/src/modules/pagination/components/pagination.component.spec.ts

```diff
@@ -174,7 +176,6 @@ describe("PaginationComponent", () => {
 			expect(pageSelectorInput.properties["id"]).toBe(currentPagePrefix + htmlSuffixId);
 			/// expect(pageSelectorInput.attributes["ngModel"]).toBe("paginationInput"); // FIXME: ngModel not included in the element
 			expect(pageSelectorInput.attributes["starkRestrictInput"]).toBe("\\d");
-			expect(pageSelectorInput.attributes["ng-reflect-on-enter-key-handler"]).toBeDefined(); // starkOnEnterKey directive
```

Comment:
Is there a way to see in the DOM whether the `keyup.enter` event is being listened?